examples/complete/main.tf

```diff
@@ -1,83 +1,65 @@
-provider "aws" {
```

Comment:
Let's not modify `examples` so aggressively. Please revert the changes in `examples/complete`.

Reply:
I rollbacked the move of the provider code sections.